woeip/apps/air_quality/factories.py

```diff
@@ -0,0 +1,74 @@
+import random
+
+import factory
+import factory.fuzzy
+from django.contrib.gis import geos
+
+from woeip.apps.air_quality.models import Data, Device, Route, Sensor, Session, SessionData
+from woeip.apps.core.factories import UserFactory
+
+
+class FuzzyLineString(factory.fuzzy.BaseFuzzyAttribute):
+    def fuzz(self):
+        points = [(random.random(), random.random())
```

Comment:
Use `factory.Faker('latlng')` to generate real coordinates. See https://faker.readthedocs.io/en/master/providers/faker.providers.geo.html for other providers.

Reply:
Is it possible to bound it to within Oakland?

Reply:
Yes, but not really worth the hassle. The models don't have any location constraints, so neither should the test data.